### PR TITLE
aio: e2e fixes

### PR DESCRIPTION
**Pull request by petebacondarwin**

### CODE REVIEW

File: aio/e2e/api.e2e-spec.ts [hunk 1]
```diff
@@ -28,7 +28,7 @@ describe('Api pages', function() {
 
   it('should show readonly properties as getters', () => {
     const page = new ApiPage('api/common/http/HttpRequest');
-    expect(page.getOverview('class').getText()).toContain('get body: T|null');
+    expect(page.getOverview('class').getText()).toContain('get body: T | null');
```
Comment (gkalpak) [inline aio/e2e/api.e2e-spec.ts]: In your face, `clang-format` 👊

File: aio/src/index.html [hunk 2]
```diff
@@ -34,11 +34,11 @@
 
   <!-- Google Analytics -->
   <script>
-  // Note this is a customised version of the GA tracking snippet to aid e2e testing
-  // See the bit between /**/.../**/
```
Comment (gkalpak) [inline aio/src/index.html]: I like this comment (and the `/**/` markers) 😞

File: aio/src/index.html [hunk 3]
```diff
@@ -34,11 +34,11 @@
...
   (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
   (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;/**/i.sessionStorage.__e2e__||/**/m.parentNode.insertBefore(a,m)
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;
+  ~i.name.indexOf('NG_DEFER_BOOTSTRAP')|| // only load library if not running e2e tests
```
Comment (gkalpak) [inline aio/src/index.html]: Where does `NG_DEFER_BOOTSTRAP` come from?

File: aio/src/index.html (hunk 2 above)
Comment (petebacondarwin) [reply aio/src/index.html]: "liked" - :-P

Trouble is that to make this comment make sense, I needed to make it longer;
but then the explanation is in the end of line comments below...

File: aio/src/index.html (hunk 3 above)
Comment (petebacondarwin) [reply aio/src/index.html]: Protractor sets it when running e2e tests :-)
I didn't want to put too much into the comments here.

Comment (gkalpak) [reply aio/src/index.html]: I wonder if they will drop it at some point (isn't it AngularJS-specific)?
I guess we can worry about that when it happens :smiley:

File: aio/src/index.html (hunk 2 above)
Comment (gkalpak) [reply aio/src/index.html]: Saying that this is acustomized version and that the changes are between `/**/` is enough for me. It makes it much easier to (a) know that this is customized and (b) what exactly has been changed. If I need more info on what it does etc, I can look up the commit.

File: aio/src/index.html (hunk 3 above)
Comment (petebacondarwin) [reply aio/src/index.html]: I understand that it is Angular as well as AngularJS.

File: aio/src/index.html (hunk 2 above)
Comment (gkalpak) [reply aio/src/index.html]: I am not approving this without my comment and/or markers :grin:

Comment (petebacondarwin) [reply aio/src/index.html]: OK OK

File: aio/src/index.html [hunk 4]
```diff
@@ -34,11 +34,13 @@
 
   <!-- Google Analytics -->
   <script>
-  // Note this is a customised version of the GA tracking snippet to aid e2e testing
-  // See the bit between /**/.../**/
+  // Note this is a customised version of the GA tracking snippet
+  // See the comments below for more info
```
Comment (petebacondarwin) [inline aio/src/index.html]: See here @gkalpak ! Comments.

**Review by gkalpak — APPROVED** [review]

**Pull request closed without merging**